dangerfile.ts

```diff
@@ -13,9 +14,9 @@ export default async () => {
   // TODO: Make sure scripts don't trigger warnings on Fabric-only PRs. Don't confuse users.
 
   // TODO: Consolidate approach with Fabric's change/checkchange functionality. Remove script implementation if unused.
-  // await checkChangelog(dangerJS);
+  await checkChangelog(dangerJS);
 
-  await detectNonApprovedDependencies(dangerJS);
+  //await detectNonApprovedDependencies(dangerJS);
```

Comment:
Revert this? Looks like a merge conflict with Jason's PR.

Reply:
Ooops yes